sale_promotion_rule_assortment/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Promotion Rule Assortment',
+    'summary': """
+        This addon allows to reduce the appliance of a promotion rule to a
+        set of products""",
+    'version': '10.0.1.0.0',
+    "development_status": "Beta",
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```

Comment:
URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)